### PR TITLE
React compatability

### PR DESCRIPTION
**Pull request by Jeddf**
- BREAKING: changing react peer dep version to ^15 from <15.5
- Applying prop-type and createClass standalone packages for react 16 compatibility

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,13 +29,15 @@
...
     "babel-preset-react": "^6.16.0",
+    "create-react-class": "^15.6.0",
     "enzyme": "^2.6.0",
     "happiness": "^7.1.2",
     "mocha": "^3.2.0",
     "mochify": "^2.18.1",
-    "react": "<15.5.0",
-    "react-addons-test-utils": "<15.5.0",
-    "react-dom": "<15.5.0"
+    "prop-types": "^15.5.10",
```
Comment (wesleytodd) [inline package.json]: I believe this should match the react version specified in the peer deps.  Not sure if it makes a difference, but I believe the publish these in lock step.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@streammedev/hermes",
-  "version": "1.4.4",
+  "version": "1.4.5-BETA",
```
Comment (wesleytodd) [inline package.json]: Preferably don't bump the version in a PR branch.  We can do the version bump with `npm version minor` straight to `1.4.5` after we merge up these PRs.

File: package.json (hunk 1 above)
Comment (Jeddf) [reply package.json]: The latest version of prop-types is 15.5.10 so I guess it's diverged from react in that regard.

File: package.json (hunk 2 above)
Comment (Jeddf) [reply package.json]: Aye! Will do. Version to 2.0.0 on master after merge per our conversation

**Review by wesleytodd — APPROVED** [review]

**Pull request merged**